lib/ash/actions/create/bulk.ex

```diff
@@ -1407,9 +1407,15 @@ defmodule Ash.Actions.Create.Bulk do
             {changesets_by_index[result.__metadata__.bulk_create_index], result}
           end)
 
+        after_batch_change_opts =
```

Comment:
Going to push up a modification to this. If the change opts aren't `{:templated, opts}` that means we need to run them *per record*, because they contain template variables. i.e `change {Change, arg(:foo)}`